doc/kshack/ainote.8

```diff
@@ -0,0 +1,182 @@
+-*- Text -*-				This is the file AI:KSHACK;AINOTE >
```

Comment:
If you're adding this document, you could also add SYSDOC; KL HUNG, KL10 FLKLOR, KL10 INFO from the GPL release (with the copyright header), and MOON; KLPROC 11 from its-vault, which give equivalent advice for the KL?

Reply:
Ok, good idea.